dev/io.openliberty.microprofile.config.2.0.internal_fat/fat/src/io/openliberty/microprofile/config/internal_fat/apps/badobserver/TestObserver.java

```diff
@@ -8,7 +8,7 @@
  * Contributors:
  *     IBM Corporation - initial API and implementation
  *******************************************************************************/
-package io.openliberty.microprofile.config.internal_fat.apps.badobserver;
+package io.openliberty.microprofile.config.internal_fat.apps.badObserver;
```

Comment:
Package name has a capitalized `O` whereas the corresponding directory does not.